aws/policy/security-services.yaml

```diff
@@ -87,6 +87,48 @@ Statement:
       - waf:UpdateSqlInjectionMatchSet
       - waf:UpdateWebACL
       - waf:UpdateXssMatchSet
+      - wafv2:ListRuleGroups
+      - wafv2:ListWebACLs
```

Comment:
WebACLs, Rules, and Requests incur fees, and I believe WAFv2 can be region restricted.  The create policies should be under a AllowRegionalRestrictedResourceActionsWhichIncurFees Sid with a resource restriction, though that will add more length to the file.  :/

Reply:
Oh Wow, pipeline succeded, there was a misplaced `"` quote.  
Now only terminator classes as missing